src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx

```diff
@@ -271,8 +271,8 @@ const StatusDisplay = ({
             }, 3000);
             console.error(e);
           } finally {
-            refresh && refresh();
-            reloadEntity && reloadEntity();
+            refresh?.();
+            reloadEntity?.();
```

Comment:
Please update the name of this file to not have the extra space before `.tsx`

Reply:
it's a copy. He was removed